Content.Server/Bible/BibleSystem.cs

```diff
@@ -81,15 +83,47 @@ private void OnAfterInteract(EntityUid uid, BibleComponent component, AfterInter
                 }
             }
 
-            var othersMessage = Loc.GetString("bible-heal-success-others", ("user", args.User),("target", args.Target),("bible", uid));
+            var othersMessage = Loc.GetString(component.LocPrefix + "-heal-success-others", ("user", args.User),("target", args.Target),("bible", uid));
             _popupSystem.PopupEntity(othersMessage, args.User, Filter.Pvs(args.User).RemoveWhereAttachedEntity(puid => puid == args.User));
 
-            var selfMessage = Loc.GetString("bible-heal-success-self", ("target", args.Target),("bible", uid));
+            var selfMessage = Loc.GetString(component.LocPrefix + "-heal-success-self", ("target", args.Target),("bible", uid));
             _popupSystem.PopupEntity(selfMessage, args.User, Filter.Entities(args.User));
 
-            SoundSystem.Play(Filter.Pvs(args.Target.Value), "/Audio/Effects/holy.ogg", args.User);
+            SoundSystem.Play(Filter.Pvs(args.Target.Value), component.HealSoundPath.GetSound(), args.User);
             _damageableSystem.TryChangeDamage(args.Target.Value, component.Damage, true);
         }
 
+        private void AddSummonVerb(EntityUid uid, BibleComponent component, GetVerbsEvent<AlternativeVerb> args)
+        {
+            if (!args.CanInteract || !args.CanAccess || !HasComp<BibleUserComponent>(args.User) || component.AlreadySummoned || component.SpecialItemPrototype == null)
+                return;
+
+            AlternativeVerb verb = new()
+            {
+                Act = () =>
+                {
+                    TransformComponent? position = Comp<TransformComponent>(args.User);
+                    AttemptSummon(component, args.User, position);
+                },
+                Text = Loc.GetString("bible-summon-verb"),
+                Priority = 2
+            };
+            args.Verbs.Add(verb);
+        }
+
+        private void AttemptSummon(BibleComponent component, EntityUid user, TransformComponent? position)
+        {
+            if (!HasComp<BibleUserComponent>(user) || component.AlreadySummoned || component.SpecialItemPrototype == null)
+                return;
+            if (!Resolve(user, ref position))
+                return;
+            if (component.Deleted || Deleted(component.Owner))
+                return;
+            if (!_blocker.CanInteract(user, component.Owner))
+                return;
+
+            EntityManager.SpawnEntity(component.SpecialItemPrototype, position.Coordinates);
+            component.AlreadySummoned = true;
+        }
```

Comment:
this should be handled by a different component than bible imo

Reply:
we've discussed in discord a lot that summons are probably a better chaplain gimmick than null rods or faith or whatever, it's intended to be one of the defining features

Reply:
i didnt say it shouldnt be on the bible